src/pyrobot/habitat/camera.py

```diff
@@ -112,15 +111,14 @@ def pix_to_3dpt(self, rs, cs, in_cam=False, initial_rotation=None):
         pts = self._cam2pyrobot(pts, initial_rotation=initial_rotation)
         return pts, colors
 
-
     def _cam2pyrobot(self, pts, initial_rotation=None):
         """
         here, points are  given in camera frame
         the thing to do next is to transform the points from camera frame into the
         global frame of pyrobot environment
         This does not translate to what habitat thinks as origin,
         because pyrobot's habitat-reference origin is `self.agent.init_state`
-        So, CAMERA frame -> HABITAT frame -> PYROBOT frame
+        So, CAMERA frame -> HABITAT frame -> PYROBOT frame (robot base frame)
```

Comment:
is this correct?

@kalyanvasudev said (when we were debugging last time) that PYROBOT frame is basically the global coordinate frame in the pyrobot environment.

The line above this also describes it differently from robot base frame:

```
the thing to do next is to transform the points from camera frame into the
        global frame of pyrobot environment
```

Reply:
Yeah, this seems right. Basically, on the real robot, the points are transformed into whatever frame set here - https://github.com/facebookresearch/pyrobot/blob/Develop/src/pyrobot/cfg/locobot_config.py#L125 which is `/base_link` i.e robot base frame.